biosys/apps/main/utils_data_package.py

```diff
@@ -54,25 +56,21 @@ def cast_any(self, value, fmt=None):
         if isinstance(value, self.python_type):
             return value
         try:
-            return date_parse(value, dayfirst=True).date()
+            # there's a 'bug' in dateutil.parser.parse (2.5.3)if you are using
+            # dayfirst=True. It will parse YYYY-MM-DD as YYYY-DD-MM !!
+            # https://github.com/dateutil/dateutil/issues/268
+            dayfirst = False if YYYY_MM_DD_REGEX.match(value) else True
```

Comment:
`dayfirst = False if YYYY_MM_DD_REGEX.match(value) else True`

could just be:

`dayfirst = not YYYY_MM_DD_REGEX.match(value)`